packages/core-components/src/components/icon/icon.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component, Prop, h } from '@stencil/core';
```

Comment:
This component should be refactored to use the proper Odyssey icon implementation.

I understand this may be out of scope for this PR. If that is the case, please remove this and any references to Icon!